.changeset/shaggy-cats-film.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": patch
+---
+
+Accept standard cookie attributes when deleting a cookie
```

Comment:
After you address matthew's review, can you update the changeset to state what new properties are available?

Reply:
Yeah we need a list of which properties we are supporting. Also this would be a `minor`.

Reply:
Updated the change log to include the final set.